### PR TITLE
Move copy_repository to internal

### PR DESCRIPTION
**Pull request by gregmagolan**
And some cleanup attached

### CODE REVIEW

File: packages/jasmine/package.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
     "name": "@bazel/jasmine",
-    "description": "Run Jasmine tests under Bazel",
+    "description": "Jasmine rules for Bazel",
+    "homepage": "https://github.com/bazelbuild/rules_nodejs",
```
Comment (alexeagle) [inline packages/jasmine/package.json]: make it a deep link to /packages/jasmine?
I have the same addition in my PR

File: packages/karma/package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@bazel/karma",
   "description": "Karma rules for Bazel",
-  "homepage": "https://github.com/bazelbuild/rules_typescript",
+  "homepage": "https://github.com/bazelbuild/rules_nodejs",
```
Comment (alexeagle) [inline packages/karma/package.json]: same

File: packages/typescript/package.json [hunk 3]
```diff
@@ -1,6 +1,7 @@
 {
     "name": "@bazel/typescript",
     "description": "TypeScript rules for Bazel",
+    "homepage": "https://github.com/bazelbuild/rules_nodejs",
```
Comment (alexeagle) [inline packages/typescript/package.json]: same

File: packages/jasmine/package.json (hunk 1 above)
Comment (gregmagolan) [reply packages/jasmine/package.json]: I'll just take that commit out and we won't have a merge conflict :)

**Pull request merged**